css/testimonial.css

```diff
@@ -0,0 +1,762 @@
+.testimonial{
```

Comment:
1.follow proper indentation
2.Don't use numbers as your class name
3.Class name should be in camel case
eg; class="thisIsMyClassName"
4.Don't define css to your html tags, always use a class name to define it.
5.Always use rounded figures in px value.Either it should end in 0's or 5's.
Update your code with this